scripts/local/e2e_test.sh

```diff
@@ -70,6 +69,12 @@ fi
 
 cd $cwd
 
+# Set the CGO flags to use the portable version of BLST
+#
+# We use "export" here instead of just setting a bash variable because we need
+# to pass this flag to all child processes spawned by the shell.
+export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
```

Comment:
Does this still need to be here now that it's included in constants.sh?

Reply:
Nope, thanks for catching.